app/models/edition/related_policies.rb

```diff
@@ -1,6 +1,16 @@
 module Edition::RelatedPolicies
   extend ActiveSupport::Concern
 
+  def delete_policy(content_id)
+    self.edition_policies.delete(EditionPolicy.where(policy_content_id: content_id))
```

Comment:
I want to remove the association between this `Edition` and its related `EditionPolicy`. Wouldn't I lose the scope if I did like you suggested?